### DISCUSSION TITLE
章节编号数字大写问题

### DISCUSSION THREAD

**Discussion by wangyizhuo777** [stone-zeng/fduthesis#333] [Q&A]
![image](https://github.com/stone-zeng/fduthesis/assets/163975097/7fc7aa72-ff18-4396-939f-ce422861c6b7)
哥，送审前又被批格式了，要把章节编号变成大写的“第一章”这种，求帮助

**Comment by sikouhjw** [comment]
https://github.com/stone-zeng/fduthesis/blob/d84e0dd7df657d47943e98c98ff843cc1336bc5f/source/fduthesis.dtx#L3851-L3864
`number      = \@@_arabic:n { chapter },` 换成 `number = \chinese{chapter},`

**Reply by syvshc** [reply]
好快 哥

**Reply by syvshc** [reply]
中文部分应该是把 
```
\keys_set:nn { ctex }
  {
    chapter =
      {
        format      = \huge \normalfont \sffamily \centering,
        beforeskip  = 50 pt,
        afterskip   = 40 pt,
        number      = \__fdu_arabic:n { chapter },    % <-- 这里的 \__fdu_arabic:n 换成 \chinese
        fixskip     = true
      },
```
